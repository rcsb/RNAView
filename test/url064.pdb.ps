%!PS-Adobe-2.0
%%Title: A postscript generated from a RNAML file.
%%CreationDate: Fri Sep 19 11:54:35 2008
%%Orientation: Portrait
%%BoundingBox:    251   213   361   580

/Times-Bold findfont 12  scalefont setfont


/Times-Bold findfont 13  scalefont setfont    %%set font for (A U G C T)

%% stacking diagram widths for base-pairs 1 & 2
/W1 {1 setlinewidth} bind def
/W2 {1.5 setlinewidth} bind def
/W3 {2 setlinewidth} bind def
/W4 {3 setlinewidth} bind def

%% minor and major grooves filling color saturation
/MINOR_SAT 0.9 def
/MAJOR_SAT 0.1 def
/OTHER_SIDES {0 0 1 sethsbcolor fill} bind def

%% defineing geometry shapes
/NP {newpath} bind def
/CIRCLE {0 360 arc closepath} bind def                        %% circle 
/TRIANGLE {moveto lineto lineto closepath} bind def           %% triangle
/SQUARE {moveto lineto lineto lineto closepath} bind def      %% square
/LINE {moveto lineto stroke} bind def                         %% line
/DASHLINE { moveto lineto [2 4] 0 setdash stroke } bind def %% dashline
/DOTLINE  {1 0 1 setrgbcolor} bind def                        %% dotline
/Dw {1 setlinewidth} bind def
/FB {setgray fill} bind def
/R6 {moveto lineto lineto lineto lineto lineto closepath} bind def
/R9 {moveto lineto lineto lineto lineto lineto
     lineto lineto lineto closepath} bind def

%% line drawing colors for ACGITU & others
/Al {0.0000 1.00 1.00 sethsbcolor} bind def
/Cl {0.1667 1.00 1.00 sethsbcolor} bind def
/Gl {0.3333 1.00 1.00 sethsbcolor} bind def
/Il {0.3333 1.00 0.57 sethsbcolor} bind def
/Tl {0.6667 1.00 1.00 sethsbcolor} bind def
/Ul {0.5000 1.00 1.00 sethsbcolor} bind def
/Xl {0.0000 0.00 0.00 sethsbcolor} bind def
/XX {0.2000 0.20 0.20 sethsbcolor} bind def
/FILLBLACK {0.0000 0.00 0.00 sethsbcolor fill} bind def

%% minor groove filling colors for ACGITU & others
/Am {0.0000 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Cm {0.1667 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Gm {0.3333 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Im {0.3333 MINOR_SAT 0.57      sethsbcolor fill} bind def
/Tm {0.6667 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Um {0.5000 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Xm {0.0000 0.00      MAJOR_SAT sethsbcolor fill} bind def

%% major groove filling colors for ACGITU & others
/AM {0.0000 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/CM {0.1667 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/GM {0.3333 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/IM {0.3333 MAJOR_SAT 0.57      sethsbcolor fill} bind def
/TM {0.6667 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/UM {0.5000 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/XM {0.0000 0.00      MINOR_SAT sethsbcolor fill} bind def

%% define line width, line join style & cap style (1 means round)
1 setlinewidth 1 setlinejoin 1 setlinecap
   271   233 translate

/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
70.00 305.17  moveto (C) center show 
70.00 276.00  moveto (C) center show 
70.00 246.83  moveto (G) center show 
70.00 217.67  moveto (A) center show 
70.00 188.50  moveto (U) center show 
70.00 159.33  moveto (G) center show 
70.00 130.17  moveto (G) center show 
70.00 101.00  moveto (U) center show 
70.00 71.83  moveto (A) center show 
70.00 42.67  moveto (G) center show 
70.00 13.50  moveto (U) center show 
70.00 -4.00  moveto (G) center show 
0.00 13.50  moveto (G) center show 
0.00 42.67  moveto (C) center show 
0.00 71.83  moveto (G) center show 
0.00 101.00  moveto (A) center show 
0.00 130.17  moveto (G) center show 
0.00 159.33  moveto (A) center show 
0.00 188.50  moveto (G) center show 
0.00 217.67  moveto (U) center show 
0.00 246.83  moveto (A) center show 
0.00 276.00  moveto (G) center show 
0.00 305.17  moveto (G) center show 
0.00 322.67  moveto (C) center show 
0.005 setlinewidth 1 setlinejoin 1 setlinecap
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 70.00 315.17 moveto (5') center show grestore
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 70.00 -14.00 moveto (3') center show grestore
Il 70.00 309.17 70.00 280.00 LINE
 Il 70.00 280.00 70.00 250.83 LINE
 Il 70.00 250.83 70.00 221.67 LINE
 Il 70.00 221.67 70.00 192.50 LINE
 Il 70.00 192.50 70.00 163.33 LINE
 Il 70.00 163.33 70.00 134.17 LINE
 Il 70.00 134.17 70.00 105.00 LINE
 Il 70.00 105.00 70.00 75.83 LINE
 Il 70.00 75.83 70.00 46.67 LINE
 Il 70.00 46.67 70.00 17.50 LINE
 Il 70.00 17.50 70.00 0.00 LINE
 /Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al -0.00 3.50 moveto (5') center show grestore
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 0.00 332.67 moveto (3') center show grestore
Tl 0.00 17.50 0.00 46.67 LINE
 Tl 0.00 46.67 0.00 75.83 LINE
 Tl 0.00 75.83 0.00 105.00 LINE
 Tl 0.00 105.00 0.00 134.17 LINE
 Tl 0.00 134.17 0.00 163.33 LINE
 Tl 0.00 163.33 0.00 192.50 LINE
 Tl 0.00 192.50 0.00 221.67 LINE
 Tl 0.00 221.67 0.00 250.83 LINE
 Tl 0.00 250.83 0.00 280.00 LINE
 Tl 0.00 280.00 0.00 309.17 LINE
 Tl 0.00 309.17 0.00 326.67 LINE
 /Times findfont 8 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Tl -11.00 217.67 moveto (103) center show grestore
gsave Tl 79.00 188.50 moveto (74) center show grestore
gsave Tl -11.00 71.83 moveto (98) center show grestore
gsave Tl 79.00 42.67 moveto (79) center show grestore
NP W1
 Xl 62.00 307.17 8.00 307.17 LINE
 Xl 62.00 311.17 8.00 311.17 LINE
 NP W1
 Xl 62.00 278.00 8.00 278.00 LINE
 Xl 62.00 282.00 8.00 282.00 LINE
 NP W1
 Xl 62.00 250.83 43.00 250.83 LINE
 Xl 38.00 247.83 38.00 253.83 43.00 250.83 TRIANGLE
 gsave  grestore stroke
 Xl 38.00 250.83 32.00 250.83 LINE
 Xl 32.00 247.83 27.00 247.83 27.00 253.83 32.00 253.83 SQUARE
 gsave  grestore stroke
 Xl 27.00 250.83 8.00 250.83 LINE
 NP W1
 Xl 62.00 221.67 43.00 221.67 LINE
 Xl 43.00 218.67 38.00 218.67 38.00 224.67 43.00 224.67 SQUARE
 gsave  grestore stroke
 Xl 38.00 221.67 32.00 221.67 LINE
 Xl 29.50 221.67 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 27.00 221.67 8.00 221.67 LINE
 NP W1
 Xl 62.00 134.17 38.00 134.17 LINE
 Xl 35.00 134.17 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 32.00 134.17 8.00 134.17 LINE
 NP W1
 Xl 62.00 105.00 43.00 105.00 LINE
 Xl 40.50 105.00 3.00 CIRCLE 
 gsave  grestore stroke
 Xl 38.00 105.00 32.00 105.00 LINE
 Xl 32.00 102.00 27.00 102.00 27.00 108.00 32.00 108.00 SQUARE
 gsave  grestore stroke
 Xl 27.00 105.00 8.00 105.00 LINE
 NP W1
 Xl 62.00 75.83 43.00 75.83 LINE
 Xl 43.00 72.83 38.00 72.83 38.00 78.83 43.00 78.83 SQUARE
 gsave  grestore stroke
 Xl 38.00 75.83 32.00 75.83 LINE
 Xl 32.00 72.83 32.00 78.83 27.00 75.83 TRIANGLE
 gsave  grestore stroke
 Xl 27.00 75.83 8.00 75.83 LINE
 NP W1
 Xl 62.00 44.67 8.00 44.67 LINE
 Xl 62.00 48.67 8.00 48.67 LINE
 NP W1
 Xl 35.00 17.50 3.00 CIRCLE 
 gsave  grestore stroke
 Al W2 64.00 192.50 6.00 192.50 gsave  DASHLINE grestore
 Al W2 64.00 163.33 6.00 163.33 gsave  DASHLINE grestore
 
 showpage 
