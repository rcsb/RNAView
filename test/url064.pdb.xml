<?xml version="1.0"?>
<!DOCTYPE rnaml SYSTEM "rnaml.dtd">

<rnaml version="1.0">

   <molecule id="1">
      <sequence>
         <numbering-system id="1" used-in-file="false">
            <numbering-range>
               <start>1</start>
               <end>12</end>
            </numbering-range>
         </numbering-system>
         <numbering-table length="12" comment="sequence number in pdb file">
           70   71   72   73   74   75   76   77   78   79 
           80   81 
         </numbering-table>
         <seq-data>
            CCGAUGGUAG UG
         </seq-data>
      </sequence>
      <structure>
         <model id="?">
            <model-info>
               <method>Crystallography ?</method>
               <resolution>? Angstroms</resolution>
            </model-info>
            <base>
               <position>1</position>
               <base-type>C</base-type>
               <atom serial="6">
                  <atom-type> O3'</atom-type>
                     <coordinates>3.881 -13.381 14.261</coordinates>
               </atom>
            </base>
            <base>
               <position>2</position>
               <base-type>C</base-type>
               <atom serial="18">
                  <atom-type> P  </atom-type>
                     <coordinates>2.365 -12.869 14.520</coordinates>
               </atom>
               <atom serial="26">
                  <atom-type> O3'</atom-type>
                     <coordinates>-1.746 -15.189 12.373</coordinates>
               </atom>
            </base>
            <base>
               <position>3</position>
               <base-type>G</base-type>
               <atom serial="38">
                  <atom-type> P  </atom-type>
                     <coordinates>-2.941 -14.194 12.843</coordinates>
               </atom>
               <atom serial="46">
                  <atom-type> O3'</atom-type>
                     <coordinates>-7.278 -15.874 15.106</coordinates>
               </atom>
            </base>
            <base>
               <position>4</position>
               <base-type>A</base-type>
               <atom serial="61">
                  <atom-type> P  </atom-type>
                     <coordinates>-8.091 -14.504 15.337</coordinates>
               </atom>
               <atom serial="69">
                  <atom-type> O3'</atom-type>
                     <coordinates>-12.822 -16.496 16.107</coordinates>
               </atom>
            </base>
            <base>
               <position>5</position>
               <base-type>U</base-type>
               <atom serial="83">
                  <atom-type> P  </atom-type>
                     <coordinates>-13.953 -15.413 16.482</coordinates>
               </atom>
               <atom serial="91">
                  <atom-type> O3'</atom-type>
                     <coordinates>-17.797 -16.596 19.651</coordinates>
               </atom>
            </base>
            <base>
               <position>6</position>
               <base-type>G</base-type>
               <atom serial="103">
                  <atom-type> P  </atom-type>
                     <coordinates>-18.361 -15.121 19.988</coordinates>
               </atom>
               <atom serial="111">
                  <atom-type> O3'</atom-type>
                     <coordinates>-19.982 -13.906 24.806</coordinates>
               </atom>
            </base>
            <base>
               <position>7</position>
               <base-type>G</base-type>
               <atom serial="126">
                  <atom-type> P  </atom-type>
                     <coordinates>-20.515 -12.394 24.600</coordinates>
               </atom>
               <atom serial="134">
                  <atom-type> O3'</atom-type>
                     <coordinates>-19.652 -8.748 28.170</coordinates>
               </atom>
            </base>
            <base>
               <position>8</position>
               <base-type>U</base-type>
               <atom serial="149">
                  <atom-type> P  </atom-type>
                     <coordinates>-20.289 -7.436 27.475</coordinates>
               </atom>
               <atom serial="157">
                  <atom-type> O3'</atom-type>
                     <coordinates>-17.855 -2.865 28.521</coordinates>
               </atom>
            </base>
            <base>
               <position>9</position>
               <base-type>A</base-type>
               <atom serial="169">
                  <atom-type> P  </atom-type>
                     <coordinates>-18.538 -1.664 27.703</coordinates>
               </atom>
               <atom serial="177">
                  <atom-type> O3'</atom-type>
                     <coordinates>-16.140 2.648 26.473</coordinates>
               </atom>
            </base>
            <base>
               <position>10</position>
               <base-type>G</base-type>
               <atom serial="191">
                  <atom-type> P  </atom-type>
                     <coordinates>-17.433 3.471 25.987</coordinates>
               </atom>
               <atom serial="199">
                  <atom-type> O3'</atom-type>
                     <coordinates>-17.497 5.950 21.315</coordinates>
               </atom>
            </base>
            <base>
               <position>11</position>
               <base-type>U</base-type>
               <atom serial="214">
                  <atom-type> P  </atom-type>
                     <coordinates>-18.973 6.536 21.038</coordinates>
               </atom>
               <atom serial="222">
                  <atom-type> O3'</atom-type>
                     <coordinates>-21.240 6.360 16.251</coordinates>
               </atom>
            </base>
            <base>
               <position>12</position>
               <base-type>G</base-type>
               <atom serial="234">
                  <atom-type> P  </atom-type>
                     <coordinates>-22.716 7.013 16.256</coordinates>
               </atom>
               <atom serial="242">
                  <atom-type> O3'</atom-type>
                     <coordinates>-26.104 6.276 12.392</coordinates>
               </atom>
            </base>
            <str-annotation>
               <single-strand>
                  <segment>
                     <seg-name>SG1</seg-name>
                     <base-id-5p><base-id><position>12</position></base-id></base-id-5p>
                     <base-id-3p><base-id><position>12</position></base-id></base-id-3p>
                  </segment>
               </single-strand>
            </str-annotation>
            <secondary-structure-display comment="x,y coodinates">
               <ss-base-coord>
                  <base-id><position>1</position></base-id>
                  <coordinates>117.857 29.464</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>2</position></base-id>
                  <coordinates>117.857 78.571</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>3</position></base-id>
                  <coordinates>117.857 127.679</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>4</position></base-id>
                  <coordinates>117.857 176.786</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>5</position></base-id>
                  <coordinates>117.857 225.893</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>6</position></base-id>
                  <coordinates>117.857 275.000</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>7</position></base-id>
                  <coordinates>117.857 324.107</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>8</position></base-id>
                  <coordinates>117.857 373.214</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>9</position></base-id>
                  <coordinates>117.857 422.321</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>10</position></base-id>
                  <coordinates>117.857 471.429</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>11</position></base-id>
                  <coordinates>117.857 520.536</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>12</position></base-id>
                  <coordinates>117.857 550.000</coordinates>
               </ss-base-coord>
            </secondary-structure-display>
         </model>
      </structure>
   </molecule>

   <molecule id="2">
      <sequence>
         <numbering-system id="2" used-in-file="false">
            <numbering-range>
               <start>1</start>
               <end>12</end>
            </numbering-range>
         </numbering-system>
         <numbering-table length="12" comment="sequence number in pdb file">
           96   97   98   99  100  101  102  103 
          104  105  106  107 
         </numbering-table>
         <seq-data>
            GCGAGAGU AGGC
         </seq-data>
      </sequence>
      <structure>
         <model id="?">
            <model-info>
               <method>Crystallography ?</method>
               <resolution>? Angstroms</resolution>
            </model-info>
            <base>
               <position>1</position>
               <base-type>G</base-type>
               <atom serial="262">
                  <atom-type> O3'</atom-type>
                     <coordinates>-26.017 -7.772 14.064</coordinates>
               </atom>
            </base>
            <base>
               <position>2</position>
               <base-type>C</base-type>
               <atom serial="277">
                  <atom-type> P  </atom-type>
                     <coordinates>-24.570 -8.386 14.469</coordinates>
               </atom>
               <atom serial="285">
                  <atom-type> O3'</atom-type>
                     <coordinates>-20.140 -6.525 12.348</coordinates>
               </atom>
            </base>
            <base>
               <position>3</position>
               <base-type>G</base-type>
               <atom serial="297">
                  <atom-type> P  </atom-type>
                     <coordinates>-19.063 -7.574 12.931</coordinates>
               </atom>
               <atom serial="305">
                  <atom-type> O3'</atom-type>
                     <coordinates>-14.795 -6.176 15.549</coordinates>
               </atom>
            </base>
            <base>
               <position>4</position>
               <base-type>A</base-type>
               <atom serial="320">
                  <atom-type> P  </atom-type>
                     <coordinates>-14.134 -7.557 16.051</coordinates>
               </atom>
               <atom serial="328">
                  <atom-type> O3'</atom-type>
                     <coordinates>-9.424 -5.364 16.786</coordinates>
               </atom>
            </base>
            <base>
               <position>5</position>
               <base-type>G</base-type>
               <atom serial="342">
                  <atom-type> P  </atom-type>
                     <coordinates>-8.122 -6.173 17.280</coordinates>
               </atom>
               <atom serial="350">
                  <atom-type> O3'</atom-type>
                     <coordinates>-4.166 -4.444 19.826</coordinates>
               </atom>
            </base>
            <base>
               <position>6</position>
               <base-type>A</base-type>
               <atom serial="365">
                  <atom-type> P  </atom-type>
                     <coordinates>-3.565 -5.924 20.043</coordinates>
               </atom>
               <atom serial="373">
                  <atom-type> O3'</atom-type>
                     <coordinates>-1.893 -7.391 24.801</coordinates>
               </atom>
            </base>
            <base>
               <position>7</position>
               <base-type>G</base-type>
               <atom serial="387">
                  <atom-type> P  </atom-type>
                     <coordinates>-1.513 -8.944 24.561</coordinates>
               </atom>
               <atom serial="395">
                  <atom-type> O3'</atom-type>
                     <coordinates>-2.928 -12.671 28.121</coordinates>
               </atom>
            </base>
            <base>
               <position>8</position>
               <base-type>U</base-type>
               <atom serial="410">
                  <atom-type> P  </atom-type>
                     <coordinates>-2.392 -14.076 27.557</coordinates>
               </atom>
               <atom serial="418">
                  <atom-type> O3'</atom-type>
                     <coordinates>-4.896 -18.505 28.416</coordinates>
               </atom>
            </base>
            <base>
               <position>9</position>
               <base-type>A</base-type>
               <atom serial="430">
                  <atom-type> P  </atom-type>
                     <coordinates>-4.028 -19.630 27.639</coordinates>
               </atom>
               <atom serial="438">
                  <atom-type> O3'</atom-type>
                     <coordinates>-6.240 -24.126 26.393</coordinates>
               </atom>
            </base>
            <base>
               <position>10</position>
               <base-type>G</base-type>
               <atom serial="452">
                  <atom-type> P  </atom-type>
                     <coordinates>-4.854 -24.891 26.031</coordinates>
               </atom>
               <atom serial="460">
                  <atom-type> O3'</atom-type>
                     <coordinates>-4.268 -27.148 21.353</coordinates>
               </atom>
            </base>
            <base>
               <position>11</position>
               <base-type>G</base-type>
               <atom serial="474">
                  <atom-type> P  </atom-type>
                     <coordinates>-2.721 -27.646 21.204</coordinates>
               </atom>
               <atom serial="482">
                  <atom-type> O3'</atom-type>
                     <coordinates>-0.557 -27.863 16.501</coordinates>
               </atom>
            </base>
            <base>
               <position>12</position>
               <base-type>C</base-type>
               <atom serial="497">
                  <atom-type> P  </atom-type>
                     <coordinates>0.960 -28.441 16.639</coordinates>
               </atom>
               <atom serial="505">
                  <atom-type> O3'</atom-type>
                     <coordinates>4.481 -27.516 12.985</coordinates>
               </atom>
            </base>
            <str-annotation>
               <single-strand>
                  <segment>
                     <seg-name>SG1</seg-name>
                     <base-id-5p><base-id><position>12</position></base-id></base-id-5p>
                     <base-id-3p><base-id><position>12</position></base-id></base-id-3p>
                  </segment>
               </single-strand>
            </str-annotation>
            <secondary-structure-display comment="x,y coodinates">
               <ss-base-coord>
                  <base-id><position>1</position></base-id>
                  <coordinates>0.000 520.536</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>2</position></base-id>
                  <coordinates>0.000 471.429</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>3</position></base-id>
                  <coordinates>0.000 422.321</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>4</position></base-id>
                  <coordinates>0.000 373.214</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>5</position></base-id>
                  <coordinates>0.000 324.107</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>6</position></base-id>
                  <coordinates>0.000 275.000</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>7</position></base-id>
                  <coordinates>0.000 225.893</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>8</position></base-id>
                  <coordinates>0.000 176.786</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>9</position></base-id>
                  <coordinates>0.000 127.679</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>10</position></base-id>
                  <coordinates>0.000 78.571</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>11</position></base-id>
                  <coordinates>0.000 29.464</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>12</position></base-id>
                  <coordinates>0.000 0.000</coordinates>
               </ss-base-coord>
            </secondary-structure-display>
         </model>
      </structure>
   </molecule>


   <interactions>
            <str-annotation>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>1</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>11</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>+</edge-5p>
                  <edge-3p>+</edge-3p>
                  <bond-orientation>c</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>2</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>10</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>+</edge-5p>
                  <edge-3p>+</edge-3p>
                  <bond-orientation>c</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>3</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>9</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>S</edge-5p>
                  <edge-3p>H</edge-3p>
                  <bond-orientation>t</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>4</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>8</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>H</edge-5p>
                  <edge-3p>W</edge-3p>
                  <bond-orientation>t</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>7</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>5</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>W</edge-5p>
                  <edge-3p>W</edge-3p>
                  <bond-orientation>t</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>8</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>4</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>W</edge-5p>
                  <edge-3p>H</edge-3p>
                  <bond-orientation>t</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>9</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>3</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>H</edge-5p>
                  <edge-3p>S</edge-3p>
                  <bond-orientation>t</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>10</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>2</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>+</edge-5p>
                  <edge-3p>+</edge-3p>
                  <bond-orientation>c</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>11</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>1</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>W</edge-5p>
                  <edge-3p>W</edge-3p>
                  <bond-orientation>c</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>5</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>7</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>!</edge-5p>
                  <edge-3p>!</edge-3p>
                  <bond-orientation>!</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>6</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>6</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>!</edge-5p>
                  <edge-3p>!</edge-3p>
                  <bond-orientation>!</bond-orientation>
               </base-pair>
               <helix id="H1">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>1</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>11</position>
                     </base-id>
                  </base-id-3p>
                  <length>11</length>
               </helix>
            </str-annotation>
   </interactions>
</rnaml>
