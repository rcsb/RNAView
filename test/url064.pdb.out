PDB data file name: url064.pdb_new
-----------------------------------------------------------
CRITERIA USED TO GENERATE BASE-PAIR: 
  3.40 --> upper H-bond length limits (ON..ON).
 26.00 --> max. distance between paired base origins.
  2.50 --> max. vertical distance between paired base origins.
 65.00 --> max. angle between paired bases [0-90].
  5.40 --> min. distance between RN9/YN1 atoms.
  8.00 --> max. distance criterion for helix break[0-12]
-----------------------------------------------------------
BASE-PAIR INSTRUCTIONS: 
Column 1 is rnaview assigned base numbers n1_n2, start from 1.
Column 2 & 3 are chain ID & residue number in input PDB file.
Column 4 is for base pair. The left & right are the bases as 
         identified by column 2 & 3 and 5 & 6.
Column 5 & 6 are residue number & chain ID in input PDB file.
Column 7 is for base pair annotation. The standard Watson-Crick
         (W.C.) pairs are annotated as -/- (AU,AT) or +/+ (GC).
         Other pairs are annotated as Leontis_Westhof Classification.
         The three edges: W(Watson-Crick); H(Hoogsteen); S(suger).
         e.g. W/H means the pair is edge of Watson-Crick & Hoogsteen.
Column 8 is glycosidic bond orientation (either cis or trans).
         e.g. 'W/H cis' means the pair is interaction on Watson-Crick
         and Hoogsteen side, glycosidic bond orientation is 'cis'.
Column 9 corresponds to Saenger Classification.

Other columns: 
        Syn sugar-base conformations are annotated as (syn).
        Stacked base pairs are annotated as (stack).
        Non-identified edges are annotated as (.) or (?)
        Tertiary interactions are marked by (!) in the line.
Reference:
Yang et al (2003) Nucleic Acids Research, Vol31,No13,p3450-3461.
-----------------------------------------------------------
BEGIN_base-pair
     1_23, A:    70 C-G   106 B: +/+ cis         XIX
     2_22, A:    71 C-G   105 B: +/+ cis         XIX
     3_21, A:    72 G-A   104 B: S/H tran        XI
     4_20, A:    73 A-U   103 B: H/W tran        XXIV
     7_17, A:    76 G-G   100 B: W/W tran        III
     8_16, A:    77 U-A    99 B: W/H tran        XXIV
     9_15, A:    78 A-G    98 B: H/S tran        XI
    10_14, A:    79 G-C    97 B: +/+ cis         XIX
    11_13, A:    80 U-G    96 B: W/W cis         XXVIII
     5_19, A:    74 U-G   102 B: W/W tran        !1H(b_b).
     6_18, A:    75 G-A   101 B: W/W cis         !1H(b_b)
END_base-pair
  The total base pairs =   9 (from   24 bases)
------------------------------------------------
 Standard  WW--cis  WW-tran  HH--cis  HH-tran  SS--cis  SS-tran
        3        1        1        0        0        0        0
  WH--cis  WH-tran  WS--cis  WS-tran  HS--cis  HS-tran
        0        2        0        0        0        2
------------------------------------------------
